src/Illuminate/Database/Eloquent/Model.php

```diff
@@ -2310,10 +2310,6 @@ public function __unset($key)
      */
     public function __call($method, $parameters)
     {
-        if (in_array($method, ['increment', 'decrement'])) {
-            return $this->$method(...$parameters);
-        }
```

Comment:
Anyone knows what the pattern for this seems to be? Why not make the actual methods public?